automation/test.sh

```diff
@@ -2,12 +2,18 @@
 
 kubectl() { cluster/kubectl.sh "$@"; }
 
-export CLUSTER_PROVIDER=$TARGET
-
 # Make sure that the VM is properly shut down on exit
 trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
 
 make cluster-down
 make cluster-up
-make cluster-sync
-make E2E_TEST_EXTRA_ARGS="-ginkgo.noColor" test/e2e/workflow
+if [ "${TEST_SUITE}" == "workflow" ]; then
+    make cluster-operator-push
+    make cluster-operator-install
+    make E2E_TEST_EXTRA_ARGS="-ginkgo.noColor" test/e2e/workflow
```

Comment:
put E2E_TEST_EXTRA_ARGS in a common env var and use it.

Reply:
Done